seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/rest/RestHttpPostCommandProcessor.java

```diff
@@ -96,6 +96,7 @@ private void handleSubmitJob(HttpPostCommand httpPostCommand, String uri)
         Config config = RestUtil.buildConfig(requestHandle(httpPostCommand));
         JobConfig jobConfig = new JobConfig();
         jobConfig.setName(requestParams.get(RestConstant.JOB_NAME));
+        CoordinatorService coordinatorService = getSeaTunnelServer().getCoordinatorService();
```

Comment:
Any reason for move this?

Reply:
Sorry, there is no need to move, there is no problem here.